### PR TITLE
Add ksh (Korn Shell) resource

### PR DESCRIPTION
**Pull request by aklyachkin**
Trivial change - added support for ksh (Korn Shell) resource.

### CODE REVIEW

File: test/unit/resources/ksh_test.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# encoding: utf-8
+# author: Christoph Hartmann
```
Comment (jerryaldrichiii) [inline test/unit/resources/ksh_test.rb]: Could you remove these author lines? We track authorship via Git so no need to replace them.

**Review by jerryaldrichiii — CHANGES_REQUESTED** [review]
Thank you for your first contribution @aklyachkin! Only a few suggestions.

In addtion to removing the author lines, could you also do DCO sign off by doing a `git commit --amend -s`? This is a little beyond a Chef's "Obvious Fix" policy I believe, see: https://docs.chef.io/community_contributions.html#the-obvious-fix-rule

For more info on DCO you can check out: https://github.com/inspec/inspec/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco

**Comment by aklyachkin** [comment]
Hi, I changed both things. Although I still think it's quite trivial - i just took bash resource and changed two letters in it ;-)

**Review by jerryaldrichiii — APPROVED** [review]

**Comment by jerryaldrichiii** [comment]
I can see that argument :smile: couldn't hurt to have the sign off though. Better safe than sorry.

**Review by jquick — APPROVED** [review]
Looks good to me! Thanks @aklyachkin

**Pull request merged**